ts/util/requests.ts

```diff
@@ -15,11 +15,11 @@ export function post(url: string, body: any) {
   });
 }
 
-export function get(base: string, body: string) {
+export function get(url: string, params: any) {
```

Comment:
`params` should have type `Record<string, string>`. This forces the keys and values to all be strings. Most other values that would be accepted as `any` would result in errors